mesh/pmesh.cpp

```diff
@@ -91,6 +91,8 @@ ParMesh::ParMesh(const ParMesh &pmesh, bool copy_nodes)
       *Nodes = *pmesh.Nodes;
       own_nodes = 1;
    }
+
+   print_shared = true;
```

Comment:
Use default initializer in the header file for default initialization.

Reply:
Good point, can you make this change?

There are probably other places in MFEM that can be simplified with default member initialization.

More generally, we should go through the code and take advantage of C++11 features like this where it makes sense.